### ISSUE TITLE
MultipleSeries plots

### ISSUE THREAD

**Issue by Mattriks** [LinkedEarth/Pyleoclim_util#154]
**Describe the bug**

- MultipleSeries plots don't seem to have a `mute=True` argument, which means I get 2 different plots (in Jupyter), e.g.

```python
S1 = pyleo.MultipleSeries(S0)

fig, ax = S1.stackplot(colors=['r', 'g', 'b', 'k' ], figsize=(4, 4))
[ax[j].set_ylim([0,1.5]) for j in np.arange(3)]
plt.show(fig)
```

- a `ylim` arg to `stackplot` would be handy

**Comment by CommonClimate** [comment]
Hi @Mattriks, thanks for bringing this to our attention. We think it would be a great way for your to get involved. Do you want to do a pull request to fix this?

**Comment by Mattriks** [comment]
Yes happy to submit a PR.  Not sure how soon I will find the time, currently dealing with lab equipment issues 😬

**Comment by CommonClimate** [comment]
Good luck with that! No urgency on our end as this is not a development priority for us.

**Comment by CommonClimate** [comment]
Bump. @Mattriks how is your Fall semester going? We're gearing up to the release of v1.0 and would like to get a sense of whether we should include this or not. Realistically it's a few months away, but any updates would be appreciated. We haven't forgotten your other issues, either!

**Comment by Mattriks** [comment]
[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is nigh, so I should try to find some time then.  Do you know you can add a "hacktoberfest" [topic](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics) to the repo?

**Comment by fzhu2e** [comment]
@Mattriks Thanks for the feedback. This is a bug that I left, so just let me fix it. :p

**Comment by khider** [comment]
@fzhu2e Let me know when you have it fixed.

**Comment by fzhu2e** [comment]
> @fzhu2e Let me know when you have it fixed.

It’s fixed in commit: https://github.com/LinkedEarth/Pyleoclim_util/commit/b4293692fa14ea1bc3b7ea6932a9e6d8a8d9e929

**Comment by fzhu2e** [comment]
> This is no longer on the master branch. Must have been overwritten by another commit (hopefully not mine!!). Will need to incorporate again in version 0.7.1

Fixed in commit 01c4419

**Comment by CommonClimate** [comment]
Thanks @fzhu2e !
